pegen/pegen.c

```diff
@@ -304,6 +304,49 @@ endmarker_token(Parser *p)
     return expect_token(p, ENDMARKER);
 }
 
+static const char *reserved_keywords[] = {
+    "and",
+    "as",
+    "assert",
+    "async",
+    "await",
+    "break",
+    "class",
+    "continue",
+    "def",
+    "del",
+    "elif",
+    "else",
+    "except",
+    "finally",
+    "for",
+    "from",
+    "global",
+    "if",
+    "import",
+    "in",
+    "is",
+    "lambda",
+    "nonlocal",
+    "not",
+    "or",
+    "pass",
+    "raise",
+    "return",
+    "try",
+    "while",
+    "with",
+    "yield"
+};
+static int // bool
+_is_reserved_keyword(char *s) {
+    for (int i = 0; i < 32; i++) {
```

Comment:
While I am okay with updating the array of reserved words manually, I am not okay with having the number of reserved words as a literal in the code. Maybe you can append a `NULL` to the array as a sentinel.